VultisigApp/VultisigApp/Chains/kujira.swift

```diff
@@ -43,9 +43,9 @@ class KujiraHelper {
         // deposit message has been set
         return try input.serializedData()
     }
-
+    
     func getPreSignedInputData(keysignPayload: KeysignPayload) throws -> Data {
-        guard case .Cosmos(let accountNumber, let sequence , let gas) = keysignPayload.chainSpecific else {
+        guard case .Cosmos(let accountNumber, let sequence , let gas, let isDeposit) = keysignPayload.chainSpecific else {
```

Comment:
same here, `isDeposit` is not used , and also it is accurate

Reply:
Sure. Done.